README.md

```diff
@@ -1,2 +1,18 @@
-# winter2022-internships
-List of Winter 2022 internships
+# Winter 2022 Internships by UW CSC
+
+Let's get this show on the road🔥🤩!! This repo has links to tech-related internships positions and external resources as well 💾.
+
+Feel free to contribute by submitting a pull request :octocat:!
```

Comment:
We may want to link the PR guide + add a form here, but we can do that in another iteration/PR it's fine. Just keep in mind this needs changing

Reply:
Sounds good!

Reply:
lgtm :) the [form](https://docs.google.com/forms/d/1WNzg99ro_jKKt3Pf35HnBUEZruIZmnJI7VdjVEzghKw/edit) is kinda ready (except for the caption) if u wanted to look it over/revise

Reply:
@dora-su If you wanted to describe a little about the repo in 1 or two sentences and then put a snippet that they could otherwise go to Github and submit a PR. That should be it for the caption.